lib/ProductOpener/Store.pm

```diff
@@ -49,6 +49,7 @@ use Storable qw(lock_store lock_nstore lock_retrieve);
 use Encode;
 use Encode::Punycode;
 use URI::Escape::XS;
+use Unicode::Normalize;
```

Comment:
Should this be added to `cpanfile` to pin it to a specific version? I'm not sure if this is distributed with core perl.

Reply:
@hangy it's a core module http://perlpunks.de/corelist/mversion?module=Unicode%3A%3ANormalize